### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Ashishgupta08**
### Video-Library App

You can check the live demo of the app [here](https://learnit-app.netlify.app/).

Please give your valuable feedback to enhance this project.

ThankYou!!!

### CODE REVIEW

**Comment by Akshay-Laxmipathi** [comment]
Your add to playlist button in the individual video pages is not working.

File: src/Pages/VideoPlayer/VideoPlayer.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                        playing='true'
+                        className="media-player"
+                    />
+                    <div className="card-text">
+                        <p className="card-primary-text">{video.title}</p>
+                        <p className="card-secondary-text">{video.views} views <b>·</b>{video.uploadDate}</p>
+                        <div className="actions">
+                            <BiLike className={like} onClick={()=>{addToLike(video)}} />
+                            <MdLibraryAdd className="action-icon" />
+                            <MdWatchLater className={watchLater} onClick={()=>{addToWatchLater(video)}} />
```
Comment (JatinRanka) [inline src/Pages/VideoPlayer/VideoPlayer.jsx]: Increase icon size

File: src/Pages/Home/Home.jsx [hunk 2]
```diff
@@ -1,22 +1,12 @@
...
-                <VideoCard />
-                <VideoCard />
-                <VideoCard />
-                <VideoCard />
-                <VideoCard />
-                <VideoCard />
-            </div> */}
             <div className="video-page">
                 {
                     videoData.map(item => <VideoCard data={item} />)
```
Comment (JatinRanka) [inline src/Pages/Home/Home.jsx]: Add key

File: src/Pages/Playlist/Playlist.jsx [hunk 3]
```diff
@@ -1,9 +1,50 @@
 import React from 'react'
+import { Nav, VideoCard } from "../../Components/index";
+import { usePlaylist } from "../../Contexts/playlist-context";
```
Comment (JatinRanka) [inline src/Pages/Playlist/Playlist.jsx]: use index file for importing

**Review by JatinRanka — CHANGES_REQUESTED** [review]

File: src/Pages/Home/Home.jsx (hunk 2 above)
Comment (Ashishgupta08) [reply src/Pages/Home/Home.jsx]: Done

File: src/Pages/Playlist/Playlist.jsx (hunk 3 above)
Comment (Ashishgupta08) [reply src/Pages/Playlist/Playlist.jsx]: Done

File: src/Pages/VideoPlayer/VideoPlayer.jsx (hunk 1 above)
Comment (Ashishgupta08) [reply src/Pages/VideoPlayer/VideoPlayer.jsx]: Done

**Pull request closed without merging**